pmd-swift/src/main/resources/category/swift/errorprone.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+
+<ruleset name="Error Prone"
+         xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
+
+    <description>
+        Rules to detect constructs that are either broken, extremely confusing or prone to runtime errors.
+    </description>
+
+    <rule name="ForceCast"
+          language="swift"
+          since="7.0"
+          message="Force casts should be avoided."
+          class="net.sourceforge.pmd.lang.rule.XPathRule"
+          externalInfoUrl="http://pmd.github.io/pmd/pmd_rules_swift_errorprone.html#forcecast">
+        <description>
+            Force casts should be avoided. This may lead to a crash if it's not used carefully.
+            For example assuming a JSON property has a given type, or your reused Cell has a certain contract.
```

Comment:
consider adding to the description detail as to what the developer should use instead